include/vrv/options.h

```diff
@@ -474,6 +474,7 @@ class Options {
     OptionInt m_pageMarginRight;
     OptionInt m_pageMarginTop;
     OptionInt m_pageWidth;
+    OptionString m_useExpansion;
```

Comment:
Could be changed to `instantiateExpansion`? or `realizeExpansion`? or `implementExpansion`?

Reply:
We had a extensive discussion with David about this. We originally used "renderExpansion", which may be confused with the toolkit render methods. Thank you for your ideas! Instantiate sounds much like coding, realize could be confused with the FRBR expression, is there really something implemented when calling this argument?

What about "expandExpansion". This is probably naming it most precisely, but sounds funny.

Reply:
> What about "expandExpansion"

Or just "expand"?

Reply:
Yes, perfect, that's the solution. Thanks for that!